libraries/standard/mqtt/test/unit/iot_tests_mqtt_api.c

```diff
@@ -1966,8 +1973,10 @@ TEST( MQTT_Unit_API, SerializeUnsubscribeChecks )
                                                 &remainingLength,
                                                 &packetSize );
     TEST_ASSERT_EQUAL_INT( IOT_MQTT_SUCCESS, status );
+    /* Make sure buffer has enough space */
+    TEST_ASSERT_GREATER_OR_EQUAL( packetSize, bufferSize );
 
-    /* Make sure subsciption count of zero fails. */
+    /* Make sure subscription count of zero fails. */
```

Comment:
While you're fixing this spelling, please remove [the misspelled word from this library's lexicon file.](https://github.com/aws/aws-iot-device-sdk-embedded-C/blob/v4_beta/libraries/standard/mqtt/lexicon.txt#L170)